django_api_forms/fields.py

```diff
@@ -137,7 +137,7 @@ def to_python(self, value) -> typing.Union[typing.Type[Enum], None]:
             try:
                 return self.enum(value)
             except ValueError:
-                raise ValidationError(f"Invalid enum value {value} passed to {type(self.enum)}")
+                raise ValidationError(f"Invalid enum value {value} passed to {self.enum}")
```

Comment:
Noticed this while looking over things again. The error it was throwing was:
```
Invalid enum value {} passed to <class 'enum.EnumMeta'>
```

This changes it to throw an error with the name of the enum, like:
```
Invalid enum value {} passed to <enum 'Color'>
```

I also fixed and uncommented the rest of those tests. Turns out I was using `FormField` in the `FormFieldList` test. I don't have any other changes planned to this PR.